proctor-common/src/test/java/com/indeed/proctor/common/TestForceGroupsOptionStrings.java

```diff
@@ -12,20 +12,20 @@ public class TestForceGroupsOptionStrings {
     static Set<String> forcePayloadTests = ImmutableSet.of("abc", "def", "xyz");
     @Test
     public void testParseForceGroupsString_Empty() {
-        assertThat(ForceGroupsOptionsStrings.parseForceGroupsString("", forcePayloadTests))
```

Comment:
i think we may need to test both with forcePayloadTests or without forcePayloadTests passed in. What do you think?

Reply:
Sounds reasonable.